nixDeps.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/Cargo.lock b/Cargo.lock
+index ee1945a..b33d791 100644
+--- a/Cargo.lock
++++ b/Cargo.lock
+@@ -3130,6 +3130,7 @@ dependencies = [
+ name = "golem-cli"
+ version = "0.0.0"
+ dependencies = [
++ "Inflector",
```

Comment:
Why is this dependency added by a patch here?

Reply:
I think I had to include it to build correctly golem-examples but I might be wrong I will check it this weekend